store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/AccumuloStore.java

```diff
@@ -412,6 +413,11 @@ protected OutputOperationHandler<GetTraits, Set<StoreTrait>> getGetTraitsHandler
         return new GetTraitsHandler(TRAITS);
     }
 
+    @Override
+    protected OperationHandler<? extends DeleteElements> getDeleteElementsHandler() {
+        return null;
```

Comment:
I know we are going to implement the accumulo logic but could we return an empty handler that does nothing instead. Just good practice. Or put a comment in to say we will revisit it.